pages/linux/xsetwacom.md

```diff
@@ -0,0 +1,23 @@
+# xsetwacom
+
+> Command line tool to change settings for Wacom pen tablets at runtime.
+
+- List all the available wacom devices. The device name is in the first column.
+
+`xsetwacom list`
+
+- Set Wacom area to specific screen. Get name of the screen with `xrandr`.
+
+`xsetwacom set "{{device name}}" MapToOutput {{screen}}`
+
+- Set mode to relative (like a mouse) or absolute (like a pen) mode.
+
+`xsetwacom set "{{device name}}" Mode "{{Relative|Absolute}}"`
+
+- Rotate the input (good for tablet-PC) by 0|90|180|270 degree from "natural" rotation.
+
+`xsetwacom set "{{device name}}" Rotate {{none|half|cw|ccw}}`
+
+- Set button to only work when Tip is touching the tablet.
```

Comment:
Wht is "Tip" capitalized? And what does it refer to? Would it make sense to write it in full, i.e. "tip of the pen/stylus"?